### DISCUSSION TITLE
Unable to connect using username and password for a couple of days.

### DISCUSSION THREAD

**Discussion by temporaldoom** [pnp/powershell#4257] [Q&A]
I'm getting the following error when trying to athenticate using connect-pnponline -interactive and using a username and password, client ID/Secret and cert still work fine.

There have been no changes to the Enterprise Registration so i'm unsure as to where the issue lays

AADSTS700016: Application with identifier '31359c7f-bd7e-475c-86db-fdb8c937548e' was not found in the directory

This is happening on all version of PNP we have in our org, 1.12.0 and 2.12.0

**Comment by temporaldoom** [comment]
https://pnp.github.io/blog/post/changes-pnp-management-shell-registration/

First time i'm hearing about, lots of scripts to rewrite (not mine)

**Reply by gautamdsheth** [reply]
Hello, you can read more about it here as well:

https://github.com/pnp/powershell/discussions/4249

**Comment by shwivel** [comment]
Screw this.
